File: Algorithms/Implementation/Birthday Chocolate/birthday.cs
//Problem URL: https://www.hackerrank.com/challenges/the-birthday-bar/problem

static int birthday(List<int> s, int d, int m) {
	int counter = 0;

	for(int i = 0; i <= s.Count() - m; i++){
		int sum = 0;

		for(int j = i; j < (i + m); j++){
			sum += s[j];
		}
		
		if(sum == d)
			counter++;
	}

	return counter;
}

File: Algorithms/Implementation/Birthday Chocolate/birthday.py
#Problem URL: https://www.hackerrank.com/challenges/the-birthday-bar/problem

def birthday(s, d, m):
    counter = 0

    for i in range(0, (len(s) - m) + 1):
        sum = 0

        for j in range(i, (i + m)):
            sum += s[j]

        if(sum == d):
            counter += 1
    
    return counter
